app/models/katello/glue/candlepin/subscription.rb

```diff
@@ -18,7 +18,7 @@ def candlepin_data(org, cp_id)
       def get_for_owner(organization)
         Katello::Resources::Candlepin::Product.all(organization).select do |product|
           #this product's id is non-numeric (marketing product), or its a custom product
```

Comment:
Is this true? Here's the IDs I see in my devel env:

```rb
Katello::Resources::Candlepin::Product.all(Organization.first.label).map { |p| [p['id'], p['name']] }
=> [["195811945746", "custom"],
 ["350", "Red Hat Ceph Storage for ARM"],
 ["230", "Red Hat Enterprise Linux 7 Server High Touch Beta"],
 ["351", "Red Hat 3scale API Management Platform"],
 ["231", "Red Hat Enterprise Linux 7 Workstation High Touch Beta"],
 ["352", "Red Hat Software Collections (for RHEL Server for ARM)"],
 ["232", "Red Hat Enterprise Linux 7 for IBM z Systems High Touch Beta"],
 ["SER0232US", "Red Hat Satellite Employee Subscription"],
...
```

It seems like (almost) all the IDs are numeric.

Reply:
Oh, I see. I didn't really understand what was meant by 'engineering product' vs. 'marketing product' but makes sense. :+1: